src/pages/manageUser.tsx

```diff
@@ -611,6 +664,87 @@ const manageUsers = () => {
             />
           </>
         )}
+
+        {/* Learners list */}
+        {value === 2 && (
+          <>
+            <Grid
+              px={'18px'}
+              spacing={2}
+              mt={1}
+              sx={{ display: 'flex', alignItems: 'center' }}
+              container
+            >
+              <Grid item xs={8}>
+                <Box>
+                  <TextField
+                    className="input_search"
+                    placeholder={t('COMMON.SEARCH_FACILITATORS')}
+                    color="secondary"
+                    focused
+                    sx={{
+                      borderRadius: '100px',
+                      height: '40px',
+                      // width: '225px',
+                    }}
+                    InputProps={{
+                      endAdornment: (
+                        <InputAdornment position="end">
+                          <SearchIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                </Box>
+              </Grid>
+              <Grid item xs={4} marginTop={'8px'}>
+                <Box>
+                  <FormControl className="drawer-select" sx={{ width: '100%' }}>
+                    <Select
+                      displayEmpty
+                      style={{
+                        borderRadius: '0.5rem',
+                        color: theme.palette.warning['200'],
+                        width: '100%',
+                        marginBottom: '0rem',
+                      }}
+                    >
+                      <MenuItem className="text-dark-grey fs-14 fw-500">
+                        {t('COMMON.FILTERS')}
+                      </MenuItem>
+                    </Select>
+                  </FormControl>
+                </Box>
+              </Grid>
+            </Grid>
+            {learnerData?.map((data: any) => (
+              <LearnersList
+                key={data.userId}
+                learnerName={data.name}
+                isDropout={data.memberStatus === 'dropout'}
```

Comment:
@suvarnakale rushikesh has added new constants for dropout in the constant.ts file please use same here

Reply:
sure